### PR TITLE
Setup configure action on tentacle :boom: :metal:

### PR DESCRIPTION
**Pull request by bigbam505**
- Sets up the tentacle if it is not already installed
- Things to note, this does not work 100% on test kitchen because of
  remote PS sessions :weary:

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -6,9 +6,9 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url 'https://github.com/cvent/octopus-deploy-cookbook'
 issues_url 'https://github.com/cvent/octopus-deploy-cookbook/issues'
-version '0.3.0'
+version '0.4.0'
 
-depends 'windows', '~> 1.38'
+depends 'windows', '~> 1.37'
```
Comment (jonathanmorley) [inline metadata.rb]: why the downgrade

File: providers/tentacle.rb [hunk 2]
```diff
@@ -74,6 +116,10 @@ def verify_version(version)
   fail 'A version is required in order to install Octopus Deploy Tentacle' unless version
 end
 
+def convert_boolean(bool)
+  bool ? 'True' : 'False'
```
Comment (jonathanmorley) [inline providers/tentacle.rb]: urgh

File: metadata.rb (hunk 1 above)
Comment (bigbam505) [inline metadata.rb]: Didn't really need to be that strict, really just need windows-service support btu they have made improvements for chef 12 that are nice

File: providers/tentacle.rb (hunk 2 above)
Comment (bigbam505) [inline providers/tentacle.rb]: Whats a better code snippet?

File: libraries/shared.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module OctopusDeploy
+  # A container to hold the shared logic instead of attributes
+  module Shared
+    def fancy_bool(bool)
+      bool.to_s.capitalize
```
Comment (bigbam505) [inline libraries/shared.rb]: @jonathanmorley its not needed as in I could just do the bool but I want to follow the .net standard

**Comment by jonathanmorley** [comment]
:shipit:

**Pull request merged**